app/controllers/suggested_councillors_controller.rb

```diff
@@ -0,0 +1,25 @@
+class SuggestedCouncillorsController < ApplicationController
+  def new
+    @suggested_councillor = SuggestedCouncillor.new
+    @authority = Authority.find(params[:authority_id])
+  end
+
+  def create
+    @suggested_councillor = SuggestedCouncillor.new(suggested_councillor_params)
+    @authority = Authority.find(params[:authority_id])
+    @suggested_councillor.authority= @authority
+
+      if @suggested_councillor.save
+        flash[:notice] = "Thank you"
+        redirect_to authority_url(@authority.short_name_encoded)
```

Comment:
good question, because I don't exactly know where it should be redirected 😅 
i wasn't sure if i should make a 'Thank you' page. I thought one I think we know and the contributor know for sure is which authority the suggested councillor belongs to, so I redirected to authority page. But it is a very loose thinking and I would love to know your suggestion!